src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -105,7 +106,7 @@ public void setDataverseType(DataverseType dataverseType) {
     private final String uncategorizedString = "Uncategorized";
 
     /**
-     * @todo Don't hard code these as English.
+     * @todo Don't hard code these as English. Use bundleUtils
```

Comment:
Yeah, all of them would be great. Sorry, GitHub only displayed a snippet of truncated code, so I only referenced those two.